test/test_webdriver.py

```diff
@@ -86,7 +86,7 @@ async def test(context):
         pass
 
     # patch with async decorator misbehaving
-    with patch('mite_selenium.Remote', autospec=True) as MockRemote:
+    with patch("mite_selenium.Remote", autospec=True) as MockRemote:
```

Comment:
Unit tests for the new api surfaces? 🙏

Reply:
Added unit tests to cover the interface changes, thank you for the help on that